pdns/auth-main.cc

```diff
@@ -117,24 +117,25 @@ AuthPacketCache PC; //!< This is the main PacketCache, shared across all threads
 AuthQueryCache QC;
 AuthZoneCache g_zoneCache;
 std::unique_ptr<DNSProxy> DP{nullptr};
-std::unique_ptr<DynListener> dl{nullptr};
+static std::unique_ptr<DynListener> g_DynListener{nullptr};
```

Comment:
Just a nit but now that the variable is local to this file, the `g_` prefix might no longer be appropriate :) `s_DynListener` ?
Same for `g_TcpNameserver`, `g_udpNameserver` and `g_udpReceivers`.

Reply:
Right, wasn't sure if they should be `s_` then. I'll change that, and maybe move some into mainthread() and out of the global scope.

Reply:
all done.